ReactAndroid/src/main/java/com/facebook/react/modules/datepicker/DatePickerDialogFragment.java

```diff
@@ -53,8 +54,44 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
     final int month = c.get(Calendar.MONTH);
     final int day = c.get(Calendar.DAY_OF_MONTH);
 
-    final DatePickerDialog dialog =
-        new DismissableDatePickerDialog(activityContext, onDateSetListener, year, month, day);
+    DatePickerMode mode = DatePickerMode.DEFAULT;  //To set mode to default
```

Comment:
nit: Please remove comment that repeats what the code does without adding information.